makefile

```diff
@@ -52,18 +52,33 @@ help:
 	@echo 'CmdStan v$(CMDSTAN_VERSION) help'
 	@echo ''
 	@echo '  Build CmdStan utilities:'
+ifeq ($(OS),Windows_NT)
+	@echo '    > mingw32-make build'
+else
 	@echo '    > make build'
+endif
 	@echo ''
 	@echo '    This target will:'
 	@echo '    1. Download the Stan compiler bin/stanc$(EXE).'
 	@echo '    2. Build the print utility bin/print$(EXE) (deprecated; will be removed in v3.0)'
 	@echo '    3. Build the stansummary utility bin/stansummary$(EXE)'
 	@echo '    4. Build the diagnose utility bin/diagnose$(EXE)'
+	@echo '    5. Build all dependencies of Stan'
 	@echo ''
 	@echo '    Note: to build using multiple cores, use the -j option to make. '
 	@echo '    For 4 cores:'
+ifeq ($(OS),Windows_NT)
+	@echo '    > mingw32-make build -j4'
+else
 	@echo '    > make build -j4'
+endif
 	@echo ''
+ifeq ($(OS),Windows_NT)
+	@echo '    On Windows it is recommended include in the PATH environment'
+	@echo '    variable the directory of the Intel TBB library.'
+	@echo '    This can be setup permanently for the user with'
```

Comment:
Suggestion:
On Windows it is recommended to include the path to the Intel TBB library int the environment variable PATH. This can be setup permanently for the user with.

My Englihs is far from perfect so feel free to ignore my comment here. The rest of it is great! Thanks for also adding all the mingw32 make instructions above.

Reply:
Will fix.